js/app.js

```diff
@@ -6,15 +6,29 @@ const Cart = function(items) {
   this.items = items;
 };
 
+
 Cart.prototype.addItem = function(product, quantity) {
   // TODO: Fill in this instance method to create a new CartItem and add it to this.items
+    let cartItem = new CartItem ();
+    Cart(cartItem);
```

Comment:
This line will create a new Cart every time the function is called. Instead of having one cart with several products, you're going to have several carts, with one product. Additionally, you're not saving the Cart anywhere, so it's going to be difficult to track.